changelog/fragments/remove-pkg-predicate.yaml

```diff
@@ -1,6 +1,6 @@
 entries:
   - description: >
-      Remove `pkg/predicate`: `ResourceFilterPredicate` is now available via
+      Removed `pkg/predicate`: `ResourceFilterPredicate` is now available via
       `pkg/ansible/predicate.NewResourceFilterPredicate()`; `GenerationChangedPredicate` has been renamed,
       modified, and moved (see this [operator-lib PR](https://github.com/operator-framework/operator-lib/pull/11)
       for details).
```

Comment:
We're going to move `pkg/ansible/predicate` to `internal/pkg/ansible/predicate`, so maybe split this into two entries?

One that says:

"Removed `pkg/predicate.ResourceFilterPredicate`"
Migration: re-implement this in your project

And another that says how to use the moved/refactored predicates in operator-lib: 

Refactored and moved `pkg/predicate.GenerationChangedPredicate`
Migration: Use `crpredicate.Or(crpredicate.GenerationChangedPredicate{}, libpredicate.NoGenerationPredicate{})` (with some details about the imports)